author: Daniel Richard G. <skunk@iSKUNK.ORG>

Avoid Rust compilation errors like

  FAILED: obj/third_party/rust/toktrie/v1/lib/libtoktrie_lib.rlib
  [...]
  error[E0658]: use of unstable library feature `unsigned_is_multiple_of`
    --> ../../third_party/rust/chromium_crates_io/vendor/toktrie-v1/src/bytes.rs:17:21
     |
  17 |     if !bytes.len().is_multiple_of(size_of::<T>()) {
     |                     ^^^^^^^^^^^^^^
     |
     = note: see issue #128101 <https://github.com/rust-lang/rust/issues/128101> for more information
     = help: add `#![feature(unsigned_is_multiple_of)]` to the crate attributes to enable
     = note: this compiler was built on 2025-03-15; consider upgrading it if it is out of date

--- a/third_party/rust/chromium_crates_io/vendor/toktrie-v1/src/lib.rs
+++ b/third_party/rust/chromium_crates_io/vendor/toktrie-v1/src/lib.rs
@@ -1,3 +1,5 @@
+#![feature(unsigned_is_multiple_of)]
+
 use serde::{Deserialize, Serialize};
 
 pub mod bytes;
--- a/third_party/rust/chromium_crates_io/vendor/llguidance-v1/src/lib.rs
+++ b/third_party/rust/chromium_crates_io/vendor/llguidance-v1/src/lib.rs
@@ -1,3 +1,5 @@
+#![feature(unsigned_is_multiple_of)]
+
 #![allow(clippy::comparison_chain)]
 #![allow(clippy::needless_range_loop)]
 
